waspc/test/Analyzer/ParserTest.hs

```diff
@@ -1,9 +1,14 @@
 module Analyzer.ParserTest where
```

Comment:
@shayneczyzewski , @craigmc08 removed old manual/unit parser tests and replaced them with golden tests, to make it easier to maintain and change the parser.
Would you pls give it a look, since you have the most experience with our current golden tests that we have so far and you can compare a bit with what you implemented before?